### PR TITLE
Shield disruptor

### PR DESCRIPTION
**Pull request by MrRowey**
Changes for #5591

### CODE REVIEW

File: .vscode/settings.json
Comment (Tagada14) [inline .vscode/settings.json:1]: Please remove this file

Comment (Garanas) [reply .vscode/settings.json:1]: no, don't remove the file. Please undo the changes 😃

Comment (Tagada14) [reply .vscode/settings.json:1]: That's what I meant XD

**Comment by Garanas** [comment]
And please wait with merging these, Gyle is casting a faf beta match today 😃

File: .vscode/settings.json
Comment (MrRowey) [reply .vscode/settings.json:1]: Opps xD

**Pull request merged**